### ISSUE TITLE
What does the document status "discontinued" mean?

### ISSUE THREAD

**Issue by rduerr** [w3c/w3process#262]
The document Delivery Context Ontology - W3C Working Group Note 29 June 2010
 available at   http://www.w3.org/TR/dcontology/ states the following:

"This document has been discontinued as part of the closure the Ubiquitous Web Applications Working Group."

This not one of the statuses in the current documentation and I am unclear on how to interpret it!

**Comment by frivoal** [comment]
The particular status this document has been retired to is "Working Group Note". This is grounded in [section 6.3.3](https://w3c.github.io/w3process/#tr-end)] of the Process.

However, as you say, it's not entirely obvious form the text in that particular specification that this is what happened. I wonder if we could make the term "discontinue" (regardless of conjugation) an anchor term in the Process that discontinued specifications could link to to explain what happened to them.

**Comment by chaals** [comment]
Note also that in 2010 the Process didn't have much about discontinuing a document - a common practice was just to abandon them. So moving the work to a Note was a pretty forward-thinking good thing to do in that case :)

**Comment by rduerr** [comment]
Yes, but it still leaves me wondering about the use of terms in that ontology, which I might otherwise be using...  The Process clearly needs to explain the term.  I note that @frivoal explanation didn't actually help me decide whether to use terms from it or not!

**Comment by chaals** [comment]
> it still leaves me wondering about the use of terms in that ontology, which I might otherwise be using

That's a question that is somewhat beyond the scope of the Process document, although I am going to give something of an answer because I suspect that @frivoal who is editing the document now might see something that could improve what the Process says:

W3C has decided that it will not continue working on the specification to make it a Recommendation.

You need to look for information about why that happened - a lack of uptake, a change of direction by implementors who had previously implemented the work, a feeling that there is no need for a standard in this area, an architectural flaw that nobody can see how to fix reasonably, a patent claim that covered essential material and meant you wouldn't have been able to use the spec royalty-free. You also want to consider whether the reason still applies. (Patents expire, dependencies on as-yet unavailable technology might be met with new technology a few years later, ...)

Then you want to consider whether what was described in the spec under development still meets your needs. Some specs are relevant for a very long time, others (for example hashing algorithms for security) clearly become inappropriate to use at some point.

Finally of course you want to look at what people are doing in the space - if they are all using a discontinued spec, or an informal modification, maybe you should propose that work re-start. If nobody is using it, maybe there is a good reason. If some do and some don't, you have to decide ...

**Comment by swickr** [comment]
+1 to what @chaals wrote.   In particular,

> W3C has decided that it will not continue working on the specification to make it a Recommendation

The text in the SoTD of that particular document might more clearly (and usefully) have been phrased in line with this as

> This document was a deliverable of the Ubiquitous Web Applications Working Group.  That group closed in 2010 without having advanced this document to Recommendation.  The public record of the group's discussions is available in the [group's archive](https://lists.w3.org/Archives/Public/public-uwa/).

**Comment by frivoal** [comment]
I'm proposing to close this issue with no change, as it's up to each individual discontinued document to explain what happened to them in their status section.

Alternatively, if we really want to be strict about this, we could add a sentence at the end of [section 6.3.3](https://w3c.github.io/w3process/#tr-end) requiring the status section to state why the document was discontinued.

**Comment by dwsinger** [comment]
Perhaps we should have 
* a definition of discontinued: "a document which the W3C is no longer actively working on and is unlikely to be revised or revived"
* a sentence that says "if a group ceases work on a document and does not expect to take it up again, the status of the document should state it is discontinued, and optionally include text explaining why"

**Comment by frivoal** [comment]
I can work on an edit along these lines if we want it. It didn't seem to me that it would really change anything concretely, and that the problem is solved by being careful when writing status section, so I am tempted to skip, but if there's consensus to do this clarification, I have no objection.

**Comment by frivoal** [comment]
@plehegar, writing this comment as a reminder: you said you wanted some time to look at this, to figure out whether you preferred closing this with no action, or favored introducing some wording in the Process to tie it to the "retired" designation that is currently used on /TR/ for such documents.

**Comment by rduerr** [comment]
As the author, I clearly would have been helped with wording in the Process that clearly explained what "discontinued" meant!

**Comment by dwsinger** [comment]
Can we simply add to the section about end-of-life , and be done?

"Any document may be marked as 'discontinued'; this means simply that the community is no longer actively working on it, and does not expect to resume."

**Comment by dwsinger** [comment]
accepted in principle, but we need to work on where this goes (there is existing text on stopping work, for example)

**Comment by dwsinger** [comment]
I suggest a light edit to [6.2.12.1](https://w3c.github.io/w3process/#abandon-draft) to add after

> Any technical report no longer intended to advance or to be maintained, and that is not being rescinded, should be published as a Working Group Note. This can happen if the Working Group decided to abandon work on the report, or the Director required the Working Group to discontinue work on the technical report before completion. If the Director closes a Working Group W3C must publish any unfinished technical report on the Recommendation track as Working Group Notes.

Such a document should be marked in its status section as (mark as definition of) 'Discontinued' accompanied by an explanation of why it was discontinued.

**Comment by plehegar** [comment]
Trying to define a definition of "discontinued" either forces to constrain the groups to this word, or forces us to add a flurry of different terms that are synonym of discontinued. Based on [today's discussion](https://www.w3.org/2020/08/26-w3process-minutes.html), it's better to add a link to the [relevant process section](https://www.w3.org/2020/Process-20200915/#abandon-draft). it was decided to abandon work and the reasons should be spelled out in the SOTD for the community to read. How those reasons are described is up to the ones making the decision and we shouldn't attempt to constrain them to specific term.

**Comment by plehegar** [comment]
The consequence of this issues is that we will:
* require the SOTD to link to the relevant process section when a document is abandoned.
* require the reason to be spelled out.

**Comment by dwsinger** [comment]
I'm fine with not making 'discontinued' a defined/required term, but merely used in its english sense, and thus with what @plehegar suggests.  So I think a sentence like:

Such a document should be documented as discontinued in its status section, with a link to this section of this document, and accompanied by an explanation of why it was discontinued.

**Comment by frivoal** [comment]
I think the proposed addition from @dwsinger is fine.

To be honest, I don't think it is necessary, and merely doing what @plehegar suggested and closing this with no fix as far as the process is concerned seems sufficient to me, but I don't oppose adding that text.

**Comment by dwsinger** [comment]
I'm pretty easy either way, but I do like people to feel that if they raise something, we handle it

**Comment by dwsinger** [comment]
Proposed Conclusion: A light edit to [6.2.12.1](https://w3c.github.io/w3process/#abandon-draft) to add after

> Any technical report no longer intended to advance or to be maintained, and that is not being rescinded, should be published as a Working Group Note. This can happen if the Working Group decided to abandon work on the report, or the Director required the Working Group to discontinue work on the technical report before completion. If the Director closes a Working Group W3C must publish any unfinished technical report on the Recommendation track as Working Group Notes.

> Such a document should be documented as **discontinued** in its status section, with a link to this section of this document, and accompanied by an explanation of why it was discontinued.

**Comment by nigelmegitt** [comment]
> Any technical report no longer intended to advance or to be maintained, and that is not being rescinded, should be published as a Working Group Note. 

@dwsinger thanks to the isolated quote, I just noticed that this wording accidentally includes  published Recs! I don't think it is meant to, from the context of the section it is in. It probably needs "unfinished" to be inserted before "technical report".

**Comment by plehegar** [comment]
> I just noticed that this wording accidentally includes published Recs! 

The title of the section says "Abandoning an Unfinished Technical Report". I'm fine with repeating it in the text but RECs aren't included in this section.

**Comment by nigelmegitt** [comment]
> The title of the section says "Abandoning an Unfinished Technical Report". I'm fine with repeating it in the text but RECs aren't included in this section.

It seems like good practice to me _not_ to depend on headings to constrain the scope of the text within them. Depending on headings is a brittle pattern; for example what might appear to be a simple editorial move of text later can accidentally have big ramifications. Better to make each normative statement stand on its own, referencing defined terms if necessary. There are other ways to deal with this, like defining document conventions but that would be a bigger change here.

So yes, I would ask for the text to repeat the word used in the heading. I'm glad you're fine with that @plehegar !

**Comment by dwsinger** [comment]
accepted, and we need a new issue on what's noticed above

**Comment by nigelmegitt** [comment]
Thanks @dwsinger , raised as #453.
